src/main/java/com/ss/ita/Menu.java

```diff
@@ -0,0 +1,217 @@
+package com.ss.ita;
+
+import com.ss.ita.util.UserName;
+import com.ss.ita.util.Scanner;
+import com.ss.ita.util.implementation.ConsoleScanner;
+
+public class Menu {
+
+    private final Scanner scanner;
+
+    //private final IRunner runner;
+
+    public Menu() {
+        //runner = new Runner();
+        scanner = new ConsoleScanner();
+    }
+
+    /**
+     * Provides list of tasks in the specified kata level
+     *
+     * @param kata kyu level of kata (from 8 to 5)
+     */
+    private static String[] tasksList(int kata) {
+        switch (kata) {
+            case 8:
+                return new String[]{
+                        "Keep Hydrated!",
+                        "Volume of a cuboid",
+                        "Miles per gallon to kilometers per liter",
+                        "To square root or no to square",
+                        "Count of positives / sum of negatives",
+                        "Convert a String to a Number!",
+                        "Willson primes",
+                        "Formatting decimal places",
+                        "Find numbers which are divisible by given number"
+                };
+
+            case 7:
+                return new String[]{
+                        "Looking for a benefactor",
+                        "Sum of the first nth term of Series",
+                        "Where is Vasya?"
+                };
+            case 6:
+                return new String[]{
+                        "Build a pile of Cubes",
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.